arrow-core-data/src/main/kotlin/arrow/core/Iterable.kt

```diff
@@ -67,13 +68,34 @@ fun <E, A> Iterable<Validated<E, Iterable<A>>>.flatSequenceValidated(semigroup:
 fun <E> Iterable<Validated<E, *>>.sequenceValidated_(semigroup: Semigroup<E>): Validated<E, Unit> =
   traverseValidated_(semigroup, ::identity)
 
+@Deprecated(
+  "In favor of zip from Kotlin std, please use zip instead of map2",
+  ReplaceWith(
+    "zip(fb, f)"
+  )
+)
 inline fun <A, B, Z> Iterable<A>.map2(fb: Iterable<B>, f: (Tuple2<A, B>) -> Z): List<Z> =
   flatMap { a ->
     fb.map { b ->
       f(Tuple2(a, b))
     }
   }
```

Comment:
This code can be removed since it didn't exist in 0.11.0. It doesn't have to be deprecated.